acceptance/tests/rototillerTask/override_command_flag_with_env_var.rb

```diff
@@ -105,4 +107,38 @@ def remove_reserved_keys(h)
       end
     end
   end
+
+  step 'Test flags that are not required' do
+    override = random_string
+    command_flags = [
+        {:name => '--not-required', :override_env => 'required_override', :required => false},
+        {:name => '--not-required-with-default', :override_env => override, :default => 'def_val', :required => false},
```

Comment:
Sorry if I am being dense but in what circumstance does the :default => 'def_val' get used?  Followup question if I set override in the ENV does it use that value?